pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -645,8 +645,18 @@ func resolveTask(
 			case errors.Is(err, remote.ErrRequestInProgress):
 				return rt, err
 			case err != nil:
+				name := pipelineTask.TaskRef.Name
+				if len(strings.TrimSpace(name)) == 0 {
+					for _, p := range pipelineTask.TaskRef.Params {
+						// TODO: amazingly, there appears to be no constants for the resolver related param names
```

Comment:
I finally found the constants for these over at https://github.com/tektoncd/pipeline/blob/main/pkg/resolution/resolver/bundle/params.go

Also found a utility function that I could reuse to get the name from the params

/hold